InventorySales/Model/CheckItemsQuantity.php

```diff
@@ -7,40 +7,54 @@
 
 namespace Magento\InventorySales\Model;
 
+use Magento\Framework\App\ObjectManager;
 use Magento\Framework\Exception\LocalizedException;
-use Magento\InventorySalesApi\Api\IsProductSalableForRequestedQtyInterface;
-use Magento\InventorySalesApi\Api\Data\ProductSalableResultInterface;
+use Magento\InventorySalesApi\Api\AreProductsSalableForRequestedQtyInterface;
 use Magento\InventorySalesApi\Api\Data\ProductSalabilityErrorInterface;
+use Magento\InventorySalesApi\Api\IsProductSalableForRequestedQtyInterface;
 
+/**
+ * Verify items are salable for requested quantity.
+ */
 class CheckItemsQuantity
 {
     /**
+     * @deprecated
+     * @see AreProductsSalableForRequestedQty
      * @var IsProductSalableForRequestedQtyInterface
      */
     private $isProductSalableForRequestedQty;
```

Comment:
You may remove private property and constructor assignment if it not used anymore.

Reply:
@sidolov, removed